pulp_rpm/app/models.py

```diff
@@ -215,6 +215,7 @@ def nvra(self):
             n=self.name, v=self.version, r=self.release, a=self.arch)
 
     class Meta:
+        default_related_name = "%(app_label)s_%(model_name)s"
```

Comment:
Can we not have a default value for this that is only overridden when necessary?

Reply:
I think for this work would be necessary to change the metaclass: https://github.com/pulp/pulpcore/blob/master/pulpcore/app/models/base.py#L40-L54
and loop through models to check if it is necessary to override the related_name.

Reply:
@daviddavis What do you think?  I know it's a late change.

Reply:
@dralley you mean [like this](https://github.com/daviddavis/pulpcore/commit/2b9ff6f4f3484bd1cf9bac09451b3484397796b2)? I raised this on the [mailing list](https://www.redhat.com/archives/pulp-dev/2019-July/msg00051.html) and [on the issue](https://pulp.plan.io/issues/4681#note-19) but only @bmbouter weighed in. I don't have a strong preference personally.

Reply:
If you want to pursue setting `default_related_name` by default, I'd recommend bumping the mailing list thread and we can go from there.